.changeset/rotten-grapes-clap.md

```diff
@@ -0,0 +1,8 @@
+---
+"@wso2is/admin.server-configurations.v1": minor
+"@wso2is/admin.extensions.v1": minor
+"@wso2is/admin.validation.v1": minor
+"@wso2is/i18n": minor
```

Comment:
since we are just adding translations to i18n module, let's bump this to patch version.

cc: @brionmario

Reply:
Changed